allocator/mimalloc-sys/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.10.0)
+
+
+project(allocator_external)
+include(ExternalProject)
+
+ExternalProject_Add(mimalloc
+  GIT_REPOSITORY    https://github.com/microsoft/mimalloc.git
+  GIT_TAG           $ENV{ALLOCATOR_MIMALLOC_TAG}
+  GIT_SHALLOW       TRUE
+  GIT_PROGRESS      TRUE
+  CONFIGURE_COMMAND ""
```

Comment:
My CMake is a little lacking, but why do all these commands just have an empty string? If they're empty, why do you need the 'USES_TERMINAL*' settings on them? (maybe some explanatory comments would be useful).